### PR TITLE
Pull Pylance with Pyright 1.1.380

### PR DESCRIPTION
**Pull request by StellaHuang95**
- Loc update
- [Handle extra path between multi root workspace roots.](https://github.com/microsoft/pylance-release/issues/6318)
- [Fix problem with settings.json watcher when multiple workspaces](https://github.com/microsoft/pylance-release/issues/6338)

### CODE REVIEW

**Review by rchiodo — APPROVED** [review]

**Comment by rchiodo** [comment]
In case it's not obvious (took me a while to find it), the difference in the mypy primer is the casing of the word 'Protocol'

File: packages/pyright-internal/src/tests/samples/typeForm4.py [hunk 1]
```diff
@@ -2,29 +2,12 @@
...
-    Required,
-    Type,
-    TypeAlias,
-    TypeGuard,
-    TypeVar,
-    Union,
-    Unpack,
-)
 import typing
+from typing import (Annotated, Any, Callable, Concatenate, Final, Generic,
```
Comment (debonte) [inline packages/pyright-internal/src/tests/samples/typeForm4.py:6]: This is strange. I'm not sure why this happened. Maybe this is an older version of typeForm4.py that we're pushing back? Maybe this is related to the `parent` commit in our `.gitrepo` being out of date?

I think we should undo this.

**Comment by debonte** [comment]
> In case it's not obvious (took me a while to find it), the difference in the mypy primer is the casing of the word 'Protocol'

Yeah, I made those changes because I felt that it was clear we were referring to a `Protocol` object not the concept and it allowed me to lock that substring, requiring the localized versions to also include `Protocol`.

Happy to back this out if there are concerns.

**Comment by rchiodo** [comment]
No concerns from me. Sounds like a good change.

File: packages/pyright-internal/src/tests/samples/typeForm4.py (hunk 1 above)
Comment (StellaHuang95) [reply packages/pyright-internal/src/tests/samples/typeForm4.py]: sure

**Review by debonte — APPROVED** [review]

**Pull request merged**